doc/classes/CubeMap.xml

```diff
@@ -84,8 +90,10 @@
 	</methods>
 	<members>
 		<member name="lossy_storage_quality" type="float" setter="set_lossy_storage_quality" getter="get_lossy_storage_quality">
+			The lossy storage quality of the CubeMap if the storage mode is set to STORAGE_COMPRESS_LOSSY.
 		</member>
 		<member name="storage_mode" type="int" setter="set_storage" getter="get_storage" enum="CubeMap.Storage">
+			The storage mode of the CubeMap.
```

Comment:
- Per the writing guidelines, use possessive:

The CubeMap's storage mode. See [code]STORAGE_*[/code] constants.